core/src/code.rs

```diff
@@ -290,78 +311,140 @@ fn check_and_canonize_gear_stack_end(module: &mut Module) -> Result<(), CodeErro
     Ok(())
 }
 
-/// Instrumentation error.
+/// Section name in WASM module.
 #[derive(Debug, PartialEq, Eq, derive_more::Display)]
-pub enum CodeError {
-    /// The provided code doesn't contain required import section.
-    #[display(fmt = "Import section not found")]
-    ImportSectionNotFound,
-    /// The provided code doesn't contain memory entry section.
+pub enum SectionName {
```

Comment:
I'd move it into its own module

Reply:
ca7304b30ae1a884fc680839cf515520bfd884a5